worker/updater.py

```diff
@@ -33,9 +33,10 @@ def update():
   zip_file.extractall(update_dir)
   zip_file.close()
   prefix = os.path.commonprefix([n.filename for n in zip_file.infolist()])
-  fishtest_src = os.path.join(update_dir, prefix)
-  fishtest_dir = os.path.dirname(worker_dir) # fishtest_dir is assumed to be parent of worker_dir
-  copy_tree(fishtest_src, fishtest_dir)
+  worker_src = os.path.join(update_dir, prefix + "worker")
```

Comment:
IMO it would read nicer and be safer to concatenate prefix and "worker" with join to not rely on prefix containing a separator. Or is there a specific reason not to use it?

Reply:
Done :)